node/protobufs/data.proto

```diff
@@ -19,8 +21,10 @@ message DataPeer {
   uint64 max_frame = 3;
   int64 timestamp = 4;
   bytes version = 5;
-  bytes signature = 6;
-  bytes public_key = 7;
```

Comment:
I dared to remove the public key as well, given that it can be derived from the peer ID. Is there any reason to keep it ?

Reply:
no, no reason now